GUI.lua

```diff
@@ -116,6 +117,14 @@ function ArenaStats:CreateGUI()
     arenaTypeDropdown:SetValue(filters.arenaType)
     asGui.f:AddChild(arenaTypeDropdown)
 
+    local nameFilter = AceGUI:Create("EditBox")
+    nameFilter:SetLabel("Filter By Name")
+    nameFilter:SetWidth(150)
+    nameFilter:SetCallback("OnEnterPressed", function(widget, event, text)
```

Comment:
How clear is it for the user that he has to press enter after typing his text? If there is not an "ok" or "search" button, I would be in favor of using "OnTextChanged" instead, of course if it's not too ressource intensive.

Reply:
It does have an "okay" button once the value in the textfield has been changed, so in my opinion it is quite obvious that enter being pressed is needed, or to click the okay button.